cli/cfncluster/cli.py

```diff
@@ -192,6 +196,22 @@ def main():
                          help='print command and exit.')
     pssh.set_defaults(func=command)
 
+    stepfunctions = subparsers.add_parser('stepfunctions',
+        description='deploy a cfncluster stepfunction via cloudformation')
+    stepfunctions.add_argument('--bucket', '-b', dest='bucket_name',
+        help='Specify s3 bucket to use/create', required=True)
+    stepfunctions.add_argument('--config', '-c', dest='config_file',
```

Comment:
Make this optional. If not specified default to `~/.cfncluster/config`

Reply:
Sounds good!